src/main/java/com/kolown/porring/security/AccountController.java

```diff
@@ -0,0 +1,29 @@
+package com.kolown.porring.security;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@AllArgsConstructor
```

Comment:
p4:
```suggestion
@RequiredArgsConstructor
```
해당 어노테이션은 어떨까요?